server/controller/RequestsController.js

```diff
@@ -101,6 +101,48 @@ class RequestsController {
       });
     } catch (error) { res.status(500).send(error.message); }
   }
+
+  /**
+ *@desc  it create creates a requests
+ *
+ * @param {string} req
+ * @param {string} res
+ *
+ * @return {object} an object
+ */
+  static async createARequest(req, res) {
+    try {
+      const {
+        requestTitle, requestBody, date, userId
+      } = req.body;
+
+      const createARequestQuery = `
+                INSERT INTO requests (
+                  request_title,
+                  request_body,
+                  date,
+                  user_id
+                )
+                VALUES (
+                  '${requestTitle}',
+                  '${requestBody}',
+                  '${date}',
+                  '${userId}'
+                ) returning *;
+              `;
+
+      const request = await client.query(createARequestQuery);
+
+      return res.status(201).json({
+        status: 'success',
+        data: {
+          request: request.rows[0]
+        }
+      });
+    } catch (error) {
+      res.status(500).send(error.message);
```

Comment:
This does not follow the JSend format. There should be consistency with your endpoint responses